### DISCUSSION TITLE
The "Help" search results are very low contrast

### DISCUSSION THREAD

**Discussion by cgay** [community/community#12302] [Accessibility]
Hi. The search results in the Help pages are extremely low contrast and difficult to read. Can you just use a fully black font there instead of light grey on white? I'm using the "Dark default" theme. See attached image.

Thanks!
<img width="1535" alt="Screen Shot 2022-03-02 at 11 32 33 AM" src="https://user-images.githubusercontent.com/491829/156405823-656fdafd-c908-4fce-9cbb-15dce4bb5f1f.png">

Chrome, Version 98.0.4758.102 (Official Build) (arm64) on macOS 12.2.1

**Comment by gaknoll** [comment]
Thanks for reporting this @cgay! I'm also able to reproduce. I'll get this shared with our docs team 🙏🏽

**Comment by cgay** [comment]
It also occurs to me that although I was reporting the immediate problem of low contrast in the Help results, another problem is that the Help results don't respect the "Default dark" theme.

**Comment by cehfisher** [comment]
Hello @cgay - thank you for providing this feedback! We wanted to follow up on this reported bug. Is this still an issue you can replicate? If it is resolved, could you please close out this issue? Thank you!

**Comment by cgay** [comment]
I dunno. Has GitHub Docs been rebranded GitHub Support? Not sure but I couldn't quickly reproduce this.

**Reply by cehfisher** [reply]
Hello @cgay - [GitHub Support](https://support.github.com) and [GitHub Docs](https://docs.github.com) are different, but related resources. I would first search for answers to your questions on Docs or this community board, then contact Support if you weren't able to find the answers you are looking for. In the case of individual account questions, you would go straight to Support. Does this info help?

**Comment by cgay** [comment]
It does help.  My problem was that I _assumed_ there would be some link to the main docs page via the menu I get by clicking on my profile picture. It has GitHub Support and GitHub Community right there, but no link to GitHub Docs. That's weird, right? And I didn't know the URL was just docs.github.com .

Mystery solved, I guess, and yes, the bug is fixed.
